bot.py

```diff
@@ -178,6 +178,18 @@ async def chirp(ctx, user=None):
     async def ruf(ctx):
         await ctx.send('https://imgur.com/q4OWXNs')
 
```

Comment:
Looks like you created an extra chirp command? Maybe change the command name to something that people can't figure out so we can continue to use it for testing purposes?

Reply:
Ah I may have failed some cleanup (I originally had a duplicate command called "chirpgpt")

Reply:
Weird, this is not what's on my local...